docs/modules/ROOT/pages/how-tos/install/cloudscale.adoc

```diff
@@ -0,0 +1,211 @@
+= Installation on Cloudscale
+
+:toc:
+
+[abstract]
+.General Procedure
+--
+. Render install manifests
+
+. Render ignition config
+
+. Provision bootstrap and control plane nodes
+
+. Provision worker and LB nodes
+
+. Synfection
+
+. Post install configuration
+--
+
+.Prerequisites
+* Cloudscale API token
+* `terraform`
+* `mc` https://docs.min.io/docs/minio-client-quickstart-guide.html[Minio client]
+* `jq`
+* Cloned https://github.com/appuio/terraform-openshift4-cloudscale[terraform-openshift4-cloudscale] repository
+
+
+== Cluster Installation
+
+. Create a new Syn cluster: https://control.vshn.net/syn/lieutenantclusters
+
+. Configure input
++
+[source,console]
+----
+export CLOUDSCALE_TOKEN=<cloudscale-token> # From https://control.cloudscale.ch/user/api-tokens
+export CLUSTER_NAME=<cluster-name>
+export BASE_DOMAIN=ocp4-poc.appuio-beta.ch
+export PULL_SECRET=<redhat-pull-secret> # From https://cloud.redhat.com/openshift/install/pull-secret
+----
+
+. Create S3 buckets
++
+[source,console]
+----
+# Use already exiting bucket user
+curl -H "Authorization: Bearer ${CLOUDSCALE_TOKEN}" \
+  https://api.cloudscale.ch/v1/objects-users | \
+  jq -e ".[] | select(.display_name == \"${CLUSTER_NAME}\")"
+
+# Or create a new one
+response=$(curl -H "Authorization: Bearer ${CLOUDSCALE_TOKEN}" \
+  -F display_name=${CLUSTER_NAME} \
+  https://api.cloudscale.ch/v1/objects-users)
+
+export AWS_ACCESS_KEY_ID=$(echo $response | jq -r .keys[0].access_key)
+export AWS_SECRET_ACCESS_KEY=$(echo $response | jq -r .keys[0].secret_key)
+
+mc config host add \
+  s3 https://objects.rma.cloudscale.ch ${AWS_ACCESS_KEY_ID} ${AWS_SECRET_ACCESS_KEY}
+
+mc mb --ignore-existing\
+  "s3/${CLUSTER_NAME}-bootstrap-ignition" \
+  "s3/${CLUSTER_NAME}-tf-state"
+
+# Ignore the error
+mc policy set download "s3/${CLUSTER_NAME}-bootstrap-ignition"
+----
+
+. Prepare `install-config.yaml`
++
+[source,console]
+----
+mkdir ${CLUSTER_NAME}
+
+cat > "${CLUSTER_NAME}/install-config.yaml" <<EOF
+apiVersion: v1
```

Comment:
If I am not mistaken, most of that config is default anyway. We have this now all over the place. Can we just create configs with the bare minimum? Does this come from the OpenShift documentation?

Reply:
Yes, this is the bare minimum required for an installation.

Reply:
I remember that in last fall, we have created clusters with only a hand full of settings. Al those master, worker and networking setting could just be ommited.